lib/file-source/src/file_server.rs

```diff
@@ -506,6 +510,12 @@ impl Default for TimingStats {
     }
 }
 
+pub enum Event {
+    Line(Line),
+    Open(FileFingerprint),
+    Close(FileFingerprint),
```

Comment:
Are we actually sending these variants anywhere?

Reply:
Oh, um, so there was a long gap between writing that enum and then finishing the work, and I thought I had covered that case. The source is also missing a test to ensure it actually handles replaced files, so I'll fix that too.

Reply:
Turns out this "event" abstraction was completely unneeded. 🤦🏻